src/fhir/2_CarePlan.DCW.Patient2.json

```diff
@@ -0,0 +1,75 @@
+{
+    "resourceType": "CarePlan",
+    "id": "CarePlan-DCW-PHQ9-GAD7",
+    "description": "Administer PHQ-9 and GAD-7",
+    "status": "active",
+    "intent": "order",
+    "category": [
+        {
+            "coding": [
+                {
+                    "system": "http://snomed.info/sct",
+                    "code": "719091000000102",
+                    "display": "Questionnaire"
+                }
+            ],
+            "text": "Questionnaire"
+        }
+    ],
+    "subject": {
+        "reference": "Patient/test-patient-2",
+        "display": "Doe, Jane"
+    },
+    "activity": [
+        {
+            "detail": {
+                "instantiatesCanonical": [
+                    "Questionnaire/CIRG-PHQ9"
```

Comment:
@mcjustin can this be generic, e.g. `Questionnaire/PHQ9` instead of CIRG specific?

Reply:
For context, this query is failing `/Questionnaire?name:contains=CIRG-PH9` when I tested this against SmartHealth IT FHIR server.

Reply:
@achen2401  I thought we agreed a few weeks ago that 1) CarePlan ... instantiatesCanonical would point at the Questionnaire's "id", and 2) that we'd prepend Questionnaires' "id" with "CIRG-".